### PR TITLE
Sending passphrase only to non users

### PR DESCRIPTION
**Pull request by leogrim**

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/emails/LibraryInviteEmailSender.scala [hunk 1]
```diff
@@ -42,14 +42,18 @@ class LibraryInviteEmailSender @Inject() (
 
       val trimmedInviteMsg = invite.message map (_.trim) filter (_.nonEmpty)
       val fromUserId = invite.inviterId
+      val passPhrase = toRecipient match {
+        case Left(_: Id[User]) => None
+        case Right(_: EmailAddress) => Some(invite.passPhrase)
+      }
```
Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/emails/LibraryInviteEmailSender.scala:48]: @eishay @stkem 
Sneaky one, it was initially:

```
val passPhrase = toRecipient match { 
   case Right(userId) => None 
   case Left(emailAddress) => Some(invite.passPhrase) 
}
```

Comment (eishay) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/emails/LibraryInviteEmailSender.scala:48]: :-)

**Comment by eishay** [comment]
thanks!

**Comment by eishay** [comment]
pelase test it in prod

**Comment by leogrim** [comment]
Tested in prod, all good.

**Comment by stkem** [comment]
:+1:

**Pull request merged**